doc/cli/npm-unpublish.md

```diff
@@ -27,6 +27,10 @@ Even if a package version is unpublished, that specific name and
 version combination can never be reused.  In order to publish the
 package again, a new version number must be used.
 
+Unpublish is only allowed with versions published in the last 24
+hours. If you are trying to unpublish a version published longer
+ago than that, contact support@npmjs.com.
```

Comment:
The `npm` client doesn't only run against `registry.npmjs.org` and private registries may have different policies when it comes to unpublishing. I think your addition here is good, but it should have the caveat that this only applies if you're using the default registry.